citar.el

```diff
@@ -115,9 +115,10 @@ for the title field for new notes."
                    :value-type string
                    :options (main suffix preview note)))
 
-(defcustom citar-insert-reference-function
-  #'citar--insert-reference
-  "A function that takes a list of (KEY . ENTRY), and returns formatted references."
+(defcustom citar-format-reference-function
+  #'citar-format-reference
+  "A function that takes a list of (KEY . ENTRY), and returns
+formatted references as a string."
```

Comment:
Do I have to set this to use the citeproc rendering? If yes, to what value?

And should this provide a choice to make this explicit?

Reply:
I added some instructions to the docstring. Hopefully makes things clearer.

Reply:
I still can't get `citar-citeproc` to autoload correctly, despite this:

```elisp
citar-format-note-function is a variable defined in ‘citar.el’.

Its value is ‘citar-citeproc-format-reference’
Original value was 
‘citar-org-format-note-default’
```